src-qt5/desktop-utils/lumina-terminal/i18n/l-terminal_el.ts

```diff
@@ -6,40 +6,40 @@
     <message>
         <location filename="../TerminalWidget.cpp" line="50"/>
         <source>Copy Selection</source>
-        <translation type="unfinished"></translation>
+        <translation>Αντιγραφή Επιλογής</translation>
     </message>
     <message>
         <location filename="../TerminalWidget.cpp" line="51"/>
         <source>Paste</source>
-        <translation type="unfinished"></translation>
+        <translation>Επικόλληση</translation>
     </message>
 </context>
 <context>
     <name>TrayIcon</name>
     <message>
         <location filename="../TrayIcon.cpp" line="123"/>
         <source>Trigger Terminal</source>
-        <translation type="unfinished"></translation>
+        <translation>Εμφάνιση/Απόκρυψη Γραμμής Εντολών</translation>
     </message>
     <message>
         <location filename="../TrayIcon.cpp" line="125"/>
         <source>Top of Screen</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>Επάνω Μέρος της Οθόνης</translation>
```

Comment:
Looks like a typo here:
The first </translation> tag in line 27 should not be there - as it should be properly closed at the end of the line instead